pipelines/graphene_nightly_pipeline

```diff
@@ -40,29 +40,29 @@ pipeline {
                 }
                 */
                 
-                stage ('Invoke Ubuntu 20.04 with 6.0 Graphene native Jobs') {
+                stage ('Invoke Ubuntu 20.04 with 6.2.8 Graphene native Jobs') {
                     steps {
-                        build job: 'local_ci_graphene_native_20.04_6.0', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
+                        build job: 'local_ci_graphene_native_20.04_6.2.8', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
                     }
                 }
 
                 stage ('Invoke Ubuntu 20.04 with 6.0 Graphene SGX Regression Suite') {
                     steps {
                         sleep(time: 55, unit: "MINUTES")
-                        build job: 'local_ci_graphene_sgx_20.04_6.0', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
+                        build job: 'local_ci_graphene_sgx_20.04_6.2.8', parameters: [[$class: 'StringParameterValue', name: 'stress_ng_run', value: 'True'], [$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
                     }
                 }
 
-                stage ('Invoke Graphene Native Suite with Debian 11 Kernel 6.0') {
+                stage ('Invoke Graphene Native Suite with Debian 11 Kernel 6.1') {
                     steps {
-                        build job: 'local_ci_graphene_native_debian_6.2', parameters: [[$class: 'StringParameterValue', name: 'gramine_repo', value: gramine_repo], [$class: 'StringParameterValue', name: 'gramine_branch', value: gramine_branch]], propagate: true, wait: true
```

Comment:
If debian was already 6.2, why we moved to 6.1

Reply:
debian had 6.0.0-0.deb11.6-amd64 before updating to v6.1. there is no kernel v6.2 available for debian yet. looks like previous job name was having incorrect version mentioned for kernel. 
Correct kernel version is mentioned for the job name.